.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node
+dist: trusty
+language: node_js
+node_js:
+  - "6.9.5"
+before_script:
+  - npm install
```

Comment:
I think when you say your project if of type `node_js` then travis will understand that you need to do `npm install`. It'll do it on your behalf and you don't need to be explicit about it by `before_script` section. 

As you see that Travis does perform this twice. It's unnecessary. https://travis-ci.org/Azure/azure-openapi-validator/builds/237738002#L139 & https://travis-ci.org/Azure/azure-openapi-validator/builds/237738002#L496